Applications/DisplaySettings/DisplaySettings.cpp

```diff
@@ -90,12 +90,11 @@ void DisplaySettingsWidget::create_wallpaper_list()
 
 void DisplaySettingsWidget::create_frame()
 {
-    m_root_widget = GUI::Widget::construct();
-    m_root_widget->load_from_gml(display_settings_window_gml);
+    load_from_gml(display_settings_window_gml);
 
-    m_monitor_widget = *m_root_widget->find_descendant_of_type_named<DisplaySettings::MonitorWidget>("monitor_widget");
+    m_monitor_widget = static_cast<DisplaySettings::MonitorWidget&>(*find_descendant_of_type_named<DisplaySettings::MonitorWidget>("monitor_widget"));
```

Comment:
You don't need to cast the return values here, that's the whole point of passing the type as a template argument to find_descendant_of_type_named() :)

Reply:
I see. So this is effectively `dynamic_cast<>()` right (if I was followign the IRC conversation correctly)